studio/components/interfaces/Settings/Logs/Logs.utils.ts

```diff
@@ -65,7 +65,7 @@ const getDotKeys = (obj: { [k: string]: unknown }, parent?: string): string[] =>
  *
  * @returns a where statement with WHERE clause.
  */
-const _genWhereStatement = (table: LogsTableName, filters: Filters) => {
+export const _genWhereStatement = (table: LogsTableName, filters: Filters) => {
```

Comment:
small nit: now that this is exported, its name shouldn't start with an underscore

Reply:
i've removed the export, we don't need to export it.